src/Theme/MarkOneTheme.ts

```diff
@@ -39,6 +39,20 @@ enum WEIGHT {
   BOLD = '600',
 }
 
+export enum VARIANT {
+  BASE = 'base',
+  INFO = 'info',
+  POSITIVE = 'positive',
+  PRIMARY = 'positive',
+  NEGATIVE = 'negative',
+  SECONDARY = 'base',
+  DANGER = 'negative',
+  DEFAULT = 'base',
+  LIGHT = 'light',
+  DARK = 'dark',
+}
```

Comment:
Should this line have a semi-colon at the end as-per eslint?

Reply:
@rmainseas When I add a semi-colon there, eslint says that it's an unnecessary semi-colon

Reply:
Interesting :thinking: Oh well, fair enough.